commands/auth.ts

```diff
@@ -73,89 +74,63 @@ type AuthArgs = CommonArgs &
 export async function handler(
   args: ArgumentsCamelCase<AuthArgs>
 ): Promise<void> {
-  const {
-    authType: authTypeFlagValue,
-    config: configFlagValue,
-    qa,
-    providedAccountId,
-  } = args;
+  const { authType: authTypeFlagValue, qa, providedAccountId } = args;
   const authType =
     (authTypeFlagValue && authTypeFlagValue.toLowerCase()) ||
     PERSONAL_ACCESS_KEY_AUTH_METHOD.value;
   setLogLevel(args);
 
   const env = qa ? ENVIRONMENTS.QA : ENVIRONMENTS.PROD;
-  // Needed to load deprecated config
-  loadConfig(configFlagValue!);
-  const configPath = getConfigPath();
-  if (configPath) {
-    checkAndWarnGitInclusion(configPath);
-  }
 
-  if (!getConfigPath(configFlagValue)) {
-    logger.error(i18n(`${i18nKey}.errors.noConfigFileFound`));
-    process.exit(EXIT_CODES.ERROR);
-  }
+  const configPath = getConfigFilePath();
```

Comment:
No longer need to manually error - `getConfigFilePath` will throw if there's no config